release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -888,6 +888,11 @@ If you customized the default OpenSSH `/etc/ssh/sshd_config` server configuratio
 
 The {cert-manager-operator} 1.11 is now generally available in {product-title} 4.14 as well as {product-title} 4.13 and {product-title} 4.12.
 
+[discrete]
+[id="ocp-4-14-ovn-k-interconnect"]
+=== Improved Scaling and Stability with Open Virtual Network (OVN) Optimizations
+{product-title} {product-version} introduces an optimization of Open Virtual Network Kubernetes (OVN-K), where its internal architecture was modified to reduce operational latency for the purposes of removing barriers to scale and performance of the networking control plane. Network flow data is now localized on every cluster node versus having a heavy control plane with centralized information. This reduces operational latency and reduces cluster-wide traffic between worker and control nodes, thus cluster networking scales linearly with node count, because additional networking capacity is added with each additional node, to optimize larger clusters. Because network flow is localized on every node, RAFT leader election of control plane nodes is no longer needed, and a primary source of instability is removed. An additional benefit to localized network flow data is that the effect of any single node loss on networking is limited to the failed node and has no bearing on the rest of the cluster’s networking, thereby making the cluster more resilient to failure scenarios. See xref:../networking/ovn_kubernetes_network_provider/ovn-kubernetes-architecture-assembly.adoc#ovn-kubernetes-architecture-assembly[OVN-Kubernetes architecture] for more information.
```

Comment:
```suggestion
{product-title} {product-version} introduces an optimization of Open Virtual Network Kubernetes (OVN-K) in which its internal architecture was modified to reduce operational latency to remove barriers to scale and performance of the networking control plane. Network flow data is now localized to cluster node instead of centralizing information on the control plane. This reduces operational latency and reduces cluster-wide traffic between worker and control nodes. As a result, cluster networking scales linearly with node count, because additional networking capacity is added with each additional node, which optimizes larger clusters. Because network flow is localized on every node, RAFT leader election of control plane nodes is no longer needed, and a primary source of instability is removed. An additional benefit to localized network flow data is that the effect of node loss on networking is limited to the failed node and has no bearing on the rest of the cluster’s networking, thereby making the cluster more resilient to failure scenarios. See xref:../networking/ovn_kubernetes_network_provider/ovn-kubernetes-architecture-assembly.adoc#ovn-kubernetes-architecture-assembly[OVN-Kubernetes architecture] for more information.
```